README.md

```diff
@@ -26,7 +26,7 @@ Also you need to configure sass loader, since all the styles are in sass format.
 | ------------------------ | ---------------- | ---------------------------------------- | ---------------------------------------- |
 | treeId                   | string           |  defaultTree                             | Tree identifier                          |
 | treeClass                | string           |  ''                                      | Tree container custom class for styling  |
-| iconClass                | string           |  ''                                      | FontAwesome content based indicators:    |
+| iconClass                | string           |  'caret'                                 | FontAwesome content based indicators:    |
```

Comment:
'caret' or 'carets'?

Reply:
Whoopsie, a typo there. It's fixed now, thanks.